hawkular-inventory-rest-api/src/main/java/org/hawkular/inventory/rest/json/RelationshipEmbeddedJacksonSerializer.java

```diff
@@ -104,7 +104,7 @@ private void serializeEntity(CanonicalPath entity, JsonGenerator jsonGenerator,
         jsonGenerator.writeStartObject();
 
         jsonGenerator.writeFieldName(FIELD_ID);
-        jsonGenerator.writeString(EntityIdUtils.getStableId(entity));
+        jsonGenerator.writeString(Inventory.getStableId(entity));
```

Comment:
Returning, `entity.toString()` could be ok. iirc, the stable id should be the same as the canonical path except for the very long canonical paths.